src/parseSection.js

```diff
@@ -8,6 +8,8 @@ const options = (section, context, argumentTemplate) => {
                 let argumentNames = '';
                 let argumentDescription = '';
 
+                const argumentEnumValues = getEnum(option, context);
+                if (argumentEnumValues[0]) argument.enum = argumentEnumValues;
```

Comment:
Maybe exist another way to validate of empty array?

Reply:
@svoboda-rabstvo yes, i have changed it.